packages/docusaurus-theme-classic/src/theme/DocBreadcrumbs/index.tsx

```diff
@@ -49,11 +54,14 @@ function BreadcrumbsItem({
 }
 
 function HomeBreadcrumbItem() {
-  const homeHref = useBaseUrl('/');
   return (
-    <BreadcrumbsItem>
-      <BreadcrumbsItemLink href={homeHref}>🏠</BreadcrumbsItemLink>
-    </BreadcrumbsItem>
+    <li className="breadcrumbs__item">
+      <Link
+        className={clsx('breadcrumbs__link', styles.breadcrumbsItemLink)}
+        href={useBaseUrl('/')}>
```

Comment:
Even though it technically works it's not a very good practice to add hooks directly in the JSX, bur rather declare all of them at the top:
- easy to see all hooks/dependencies being used by this components
- no potential error if you start to do conditional rendering upper in the tree

Reply:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/749374/154480472-2e2bd881-036f-46b4-8ef6-97ce40526201.png">